rllib/utils/replay_buffers/utils.py

```diff
@@ -179,7 +192,6 @@ def validate_buffer_config(config: dict):
         "prioritized_replay_beta",
         "prioritized_replay_eps",
         "no_local_replay_buffer",
-        "replay_batch_size",
```

Comment:
Can we leave a warning here if users are using this setting? Happy to make this an error, but we should produce some meaningful explanation as to what happened to this setting.

Reply:
It's been a warning only so far, but per your request I've made it an error.